### ISSUE TITLE
Need any support?

### ISSUE THREAD

**Issue by bmschmidt** [machines-reading-maps/deepscatter-experiments#1]
Github recommended this as a repo that I 'might be interested in,' and indeed I am--reach out if there's any support, preview features, etc. I might offer.

**Comment by rsimon** [comment]
Ha ha - well thanks for the offer 😃. I'll keep you posted and reach out when I get stuck. But using deepscatter, so far, looks too straightforward and usable to get stuck ;-)

I only tried the pre-packaged sample/random test dataset quickly yesterday though. Next step will be to display all the placenames we harvested from our historic maps. Around 10 million, and all have x/y coordinates in Web Mercator projection, which should be usable out of the box I think. Will probably try this tomorrow (and post results here).

**Comment by rsimon** [comment]
Ok, I do have a problem (which is probably a newbie issue) after all. 

- I prepared our data as a CSV with three columns: `x`, `y` and `map_id` (the map ID is something I might want to do color coding on later)
- I ran the `quadfeather` utility to generate the data tiles. Seems to have worked and gave me a folder with zoom levels 0 to 4.
- I copied the tiles folder into the `deepscatter` folder and ran `npm install`, then `npm run dev`
- The browser shows the demo/test page, but without data. The console log is repeating the following error message:

```
regl_rendering.ts:262 Error: Requested quantity but table only has columns ["x", "y", "map_id", "ix]"
``` 

Any ideas what I might be doing wrong?

**Comment by bmschmidt** [comment]
Yeah, try searching for `quantity` in the index.html. The chart specification in `encoding` is calling for it on one of the visual dimensions here: https://github.com/nomic-ai/deepscatter/blob/main/index.html#L65-L78

If you delete those lines (the entry in the Object for 'size') it ought to run. (I'd also delete the lines about jitter_radius--I should pull those out of the demo file, because they help with debugging but aren't what people likely want to see.)

**Comment by rsimon** [comment]
Thanks!

Re 1: yes, search on the labels would be fantastic. I wouldn't have expected that this is still feasible with 10 million data points! I may not have the time to try this immediately. But let's definitely chat about it.

Re 2: can you explain what you mean by "normalized coordinates"?

Re 3: wow - yes, that would indeed be amazing. (And I'm sure @kmcdono2 would love it!). Is there an example how I can build such a transition? And how does this work? Do I need to pre-generate a sequence of tilesets for the animation? Or is there some realtime magic going on the browser?
